documentation/userguide.md

```diff
@@ -415,6 +415,12 @@ By default, the upload daemon will not validate SSL certificates. We strongly re
 By default, the upload daemon will upload finished trace files every 5 minutes. To change this interval, set the `uploadIntervalInMinutes` option in the config file to the desired value.
 Set the option to `0` to disable scheduled uploads. The uploader will then upload finished trace files when invoked and terminate immediately after. This allows complete manual control of uploads.
 
+## Disabling Trace File Merging
```

Comment:
I would call this "Trace-File Merging" (because that's the feature).

Reply:
AFAIK English doesn't use a hyphen here. But I'm not 100% sure if you meant "insert a hyphen between trace and file" with your comment or "change the headline to 'trace file merging' instead of 'disabling trace file merging'"?

Reply:
Both, actually ;)

I think the section should describe the feature, including how to disable it. As a user, I want the guide to tell me what the tool can do. Therefore, I would remove the "Disabling" in the header.

The hyphen is required at this point, because otherwise it would mean that there is a file merging that you trace, as opposed to a trace file that your merge. This is because [english is right-associative](https://www.bodden.de/2008/07/16/english-is-right-associative-french-is-not/).